static/_redirects

```diff
@@ -26,6 +26,18 @@
 /flagger/usage/kuma-progressive-delivery        /flagger/tutorials/kuma-progressive-delivery        301!
 /flagger/usage/gatewayapi-progressive-delivery  /flagger/tutorials/gatewayapi-progressive-delivery  301!
 
+/flux/installation/#bootstrap                                         /flux/installation/bootstrap                             301!
```

Comment:
@mehak151 do these actually work? Can you redirect a HTML anchor?

Reply:
My guess is that these redirects will not allow people to see the installation page at all.

Reply:
@stefanprodan These redirects take the user to the installation page instead of the bootstrap page. The hash part is not taken into account.

Reply:
So we broke installation, people will never see it. All these have to be undone and we only keep the cloud guides redirects.

Reply:
There is no bootstrap page in the old docs. Is the other way around, people will not be able to access /installation at all.

Reply:
People can still access the installation page, but they won't be able to find the bootstrap section in it.
Here's the build: https://delicate-biscotti-c79aa9.netlify.app/flux/installation/#bootstrap